src/components/FancyValue/FancyValue.tsx

```diff
@@ -36,17 +36,27 @@ const FancyValue: React.FC<FancyValueProps> = ({ icon, label, value, valueSize,
 
   const DisplayHint = useMemo(() => {
     if (hint) {
+      let isNum: boolean;
+      //if hint means price Change
+      if (hint.charAt(hint.length - 1) == "%") {
+        isNum = true;
+      }
+      //hint means normal info
+      else {
+        isNum = false;
+      }
+      console.log(isNum);
```

Comment:
no need for this here, just move the `isNum` to be a param on FancyValue, to be controllable manually anywhere without restricting it to any "%" symbol